### PR TITLE
Add neptune

### PR DESCRIPTION
**Pull request by paulray**
Add neptune position to TOAs table and to Solar System Shapiro calculation.

### CODE REVIEW

File: src/pint/toa.py [hunk 1]
```diff
@@ -1638,7 +1638,7 @@ def compute_posvels(self, ephem=None, planets=False):
             if c in self.table.colnames:
                 log.info("Column {0} already exists. Removing...".format(c))
                 self.table.remove_column(c)
-        for p in ("jupiter", "saturn", "venus", "uranus"):
+        for p in ("jupiter", "saturn", "venus", "uranus", "neptune"):
```
Comment (aarchiba) [inline src/pint/toa.py:1641]: Can this tuple go up top in a variable so it doesn't have to be repeated all these times?

**Comment by aarchiba** [comment]
Sounds like the Enterprise folks want all the planets?

**Comment by paulray** [comment]
There will be a storage and time cost to adding all the planets, but I guess it will only be paid if PLANET_SHAPIRO is true?

**Comment by aarchiba** [comment]
Sounds like they're not very interested.

**Pull request merged**